crates/turbopack-tests/tests/execution/turbopack/tree-shaking/basic/input/index.js

```diff
@@ -0,0 +1 @@
+import "../../../side-effects-optimization/basic/input/index.js";
```

Comment:
@sokra Do you have any idea how can the current changes cause the resolving difference?